packages/devtools-launchpad/src/index.js

```diff
@@ -180,10 +180,13 @@ async function getTabs(actions) {
 async function bootstrap(React, ReactDOM) {
   const connTarget = getTargetFromQuery();
   if (connTarget) {
-    const { tab, tabConnection } = await startDebugging(connTarget);
+    const debuggedTarget = await startDebugging(connTarget);
 
-    await updateConfig();
-    return { tab, connTarget, tabConnection };
+    if (debuggedTarget) {
```

Comment:
maybe we want to show something if debuggedTarget is falsy ?

Reply:
Was thinking we could log an information to console that the tab has been closed because as a result of the debugged source having been closed.